pandas/plotting/_core.py

```diff
@@ -2721,18 +2721,82 @@ def line(self, x=None, y=None, **kwds):
 
     def bar(self, x=None, y=None, **kwds):
         """
-        Vertical bar plot
+        Vertical bar plot.
+
+        A bar plot is a plot that presents categorical data with
+        rectangular bars with lengths proportional to the values that they
+        represent. A bar plot shows comparisons among discrete categories. One
+        axis of the plot shows the specific categories being compared, and the
+        other axis represents a measured value.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, default None
+            Allows plotting of one column versus another.
+        y : label or position, default None
+            Allows plotting of one column versus another.
+        **kwds : optional
```

Comment:
In our case it worked with `kwds` instead of `**kwds`. 
https://github.com/pandas-dev/pandas/pull/20152/files#diff-2b118bda866f3d626ceb6b529c62cd1aR1977